Add physical types to module namespace (#11691)
astropy/astropy: docs/changes/units/11691.feature.rst

> Each physical type is added to ``astropy.units.physical``.

Reviewer: I'd also given an example here: "(e.g., ``physical.length``)."

> Each physical type is added to the physical module in ``units`` along with a

Reviewer 2: ```suggestion
Each physical type is added to ``astropy.units.physical`` along with a
```
Slight edit.  Another possibility would be to add back ticks around "physical" in "physical module".
